src/map/map.hpp

```diff
@@ -66,11 +66,15 @@ void map_msg_reload(void);
 #define NATURAL_HEAL_INTERVAL 500
 #define MIN_FLOORITEM 2
 #define MAX_FLOORITEM START_ACCOUNT_NUM
-#define MAX_LEVEL 250
 #define MAX_DROP_PER_MAP 48
 #define MAX_IGNORE_LIST 20 	// official is 14
 #define MAX_VENDING 12
 #define MAX_MAP_SIZE 512*512 	// Wasn't there something like this already? Can't find it.. [Shinryo]
+#ifdef RENEWAL
```

Comment:
Splitting would require more adjustments. For example on the setting
https://github.com/rathena/rathena/blob/master/conf/battle/homunc.conf#L53

The max level for each job is capped in job_exp.yml anyway, revert this.